### PR TITLE
Updated Fort.js to v2.0.0 and the homepage, author info

### PR DESCRIPTION
**Pull request by idriskhenchil**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/idriskhenchil/Fort.js
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 864
   - Count of watchers: 36
   - Count of forks: 98
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/Fort.js/package.json [hunk 1]
```diff
@@ -18,4 +16,4 @@
     "type": "git",
     "url": "git://github.com/idriskhenchil/Fort.js"
   }
-}
+}
```
Comment (PeterDaveHello) [inline ajax/libs/Fort.js/package.json]: Please don't touch the last line, thanks.

**Comment by PeterDaveHello** [comment]
@idriskhenchil is it possible to apply auto-update config on your lib?

File: ajax/libs/Fort.js/package.json [hunk 2]
```diff
@@ -2,12 +2,10 @@
   "name": "Fort.js",
   "title": "Fort.js",
   "filename": "fort.min.js",
-  "version": "1.0.0",
+  "version": "2.0.0",
   "description": "Modern progress bar for form completion.",
-  "homepage": "http://colourity.github.io/",
```
Comment (PeterDaveHello) [inline ajax/libs/Fort.js/package.json]: Doesn't this work anymore?

Comment (idriskhenchil) [inline ajax/libs/Fort.js/package.json]: @PeterDaveHello No, that link no longer works

Comment (PeterDaveHello) [inline ajax/libs/Fort.js/package.json]: Okay, would you please add the newline char back? Thanks.

Comment (idriskhenchil) [inline ajax/libs/Fort.js/package.json]: Added! @PeterDaveHello

**Pull request merged**